src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,116 @@
-import React from 'react';
+import React, { ChangeEvent, useCallback, useState } from 'react';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { MovieCard } from '../MovieCard';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+interface Props {
+  isMovieInTheList: (movie: Movie) => boolean;
+  addMovie: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ isMovieInTheList, addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const findMovie = useCallback(async () => {
+    if (!query.trim()) {
+      setErrorMessage('Enter search text');
+    }
+
+    if (query.trim()) {
+      setErrorMessage('');
+      const request = await getMovie(query);
+
+      if (request.Response === 'True') {
+        setMovie(request);
+        setErrorMessage('');
+      }
+
+      if (request.Response === 'False') {
+        setErrorMessage('Can\t find a movie with such title');
+      }
+    }
+  }, [errorMessage, query]);
+
+  const addMovieToTheList = useCallback(() => {
+    if (movie) {
```

Comment:
Same here. Move `if (!movie)` at the beginning and add return, then you will not need this `if`